### PR TITLE
[Zip] New package: 99natmar99.Windows11Fixer version 2.1.0

### LINKED ISSUE #88057: [Package Request]: Windows 11 Fixer
### Package Requested

- [ ] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [ ] I have checked that the installer for this package is not contained in a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: 99natmar99
Publisher Url: https://github.com/99natmar99
Package Name: Windows 11 Fixer

Description: Windows 11 Fixer is a program designed to make customizing your Windows 11 as easy as possible. Windows 11 Fixer provides one easy to use location to customize Windows 11, change a large amount of Windows settings (including security settings), uninstall undesired Windows software, and install most of the programs you'll want or need.

Package Version: v2.1.0
Installer URL: https://github.com/99natmar99/Windows-11-Fixer/releases/tag/v2.1.0

Current Package Identifier (for new versions): -

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

* Resolves #88057

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88164)

### CODE REVIEW

**Comment by Trenly** [comment]
@msftbot .zip

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
Ping-pong @stephengillie

**Comment by stephengillie** [comment]
Validation is failing because the pipeline Current OS is lower than the MinOSVersion of 10.0.22621.

File: manifests/9/99natmar99/Windows11Fixer/2.1.0/99natmar99.Windows11Fixer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: 99natmar99.Windows11Fixer
+PackageVersion: 2.1.0
+MinimumOSVersion: 10.0.22621
+InstallerType: zip
+ReleaseDate: 2022-03-09
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: ./Windows 11 Fixer v2.1.0 (Portable)/Windows 11 Fixer.exe
```
Comment (stephengillie) [inline manifests/9/99natmar99/Windows11Fixer/2.1.0/99natmar99.Windows11Fixer.installer.yaml:13]:
```suggestion
  - RelativeFilePath: \Windows 11 Fixer v2.1.0 (Portable)\Windows 11 Fixer.exe
```

**Comment by stephengillie** [comment]
Hi @Trenly,

The installer did not complete:
2023-03-09 17:04:27.234 [CLI ] Extracting archive to: C:\Users\User\AppData\Local\Temp\WinGet\99natmar99.Windows11Fixer.2.1.0\extracted
2023-03-09 17:04:29.919 [CLI ] Successfully extracted archive
2023-03-09 17:04:29.920 [CLI ] Unable to locate nested installer at: C:\Users\User\AppData\Local\Temp\WinGet\99natmar99.Windows11Fixer.2.1.0\extracted\./Windows 11 Fixer v2.1.0 (Portable)/Windows 11 Fixer.exe
2023-03-09 17:04:29.921 [CLI ] Terminating context: 0x8a15005b at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\ArchiveFlow.cpp:62


It was unable to locate the nested installer. I think it might have an issue with the RelativeFilePath,

File: manifests/9/99natmar99/Windows11Fixer/2.1.0/99natmar99.Windows11Fixer.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/9/99natmar99/Windows11Fixer/2.1.0/99natmar99.Windows11Fixer.installer.yaml:13]:
```suggestion
  - RelativeFilePath: Windows 11 Fixer v2.1.0 (Portable)/Windows 11 Fixer.exe
```

**Comment by stephengillie** [comment]
Hi @Trenly,

The package won't run on 64-bit OSes: 
```
The program or feature "\??\C:\Users\User\AppData\Local\MICROS~1\WinGet\Links\WINDOW~1.EXE" cannot start or run due to incompatibity with 64-bit versions of Windows. Please contact the software vendor to ask if a 64-bit Windows compatible version is available.
```

Also both Explorer and PowerShell did not have an associated application to run it: 
image

**Comment by stephengillie** [comment]
Closing with reason: No recent activity.

**Pull request closed without merging**